README.md

```diff
@@ -62,12 +82,6 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] You guys have done a great job. You have added a link here to the license file of this project which is broken because you forgot to add the license file. :smile: Please add this file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: